pkg/k8shandler/fluentd.go

```diff
@@ -337,6 +354,41 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateFluentdSecret() error
 	return nil
 }
 
+func newLogMetricExporterContainer() v1.Container {
+	collectionSpec := logging.CollectionSpec{}
+        if cluster.Spec.Collection != nil {
+                collectionSpec = *cluster.Spec.Collection
+        }
+	resources := collectionSpec.Logs.LogFileMetricExporterSpec.Resources 
+        if resources == nil {
+                resources = &v1.ResourceRequirements{
+                        Limits: v1.ResourceList{v1.ResourceMemory: defaultLogFileMetricExporterMemory},
+                        Requests: v1.ResourceList{
+                                v1.ResourceMemory: defaultLogFileMetricExporterMemory,
+                                v1.ResourceCPU:    defaultLogFileMetricExporterCpuRequest,
+                        },
+                }
+        }
+
+
+	container := NewContainer(constants.LogMetricExporterName, constants.LogMetricExporterName, v1.PullIfNotPresent, *resources)
+	container.Command = []string{"log-file-metric-exporter"}
```

Comment:
this command takes arguments right? why not make it clear that we are only watching application logs by giving an argument `--dir /var/log/containers` 
Since we are mounting complete `/var/log` in the container, it is not clear what are we watching for. giving argument makes it clear, even though that is the default argument

Reply:
if we want to watch only app logs, then mount the `/var/loc/containers` to be more explicit about it.

Reply:
@jcantrill @alanconway reverted back log-file-metric-exporter from internal\pkg, internal\cmd, taken out resource spec defined specifically for this exporter. Repo is 99% established in here for building , testing log-file-metric-exporter as an independent component - https://github.com/ViaQ/log-file-metric-exporter/tree/master. kindly review all my changes in this PR.